odoo/addons/base/models/res_country.py

```diff
@@ -150,12 +151,23 @@ def _check_address_format(self):
                 except (ValueError, KeyError):
                     raise UserError(_('The layout contains an invalid format key'))
 
+    @api.depends('country_group_ids')
+    def _compute_country_group_codes(self):
+        for country in self:
+            country_group_codes = [
+                # Check if there is no False value
+                country_group.code
+                for country_group in country.country_group_ids
+                if country_group.code
+            ]
+            country.country_group_codes = ",".join(country_group_codes)
```

Comment:
```suggestion
            country.country_group_codes = ",".join(["", *country_group_codes, ""]) if country_group_codes else ""
```

Reply:
@william-andre How difficult would it be to just have country_group_codes as a json field and use this as a list directly?